crutest/src/cli.rs

```diff
@@ -1071,9 +1100,12 @@ pub async fn start_cli_server(
      * If write_log len is zero, then the RegionInfo has
      * not been filled.
      */
+    let volume_info = VolumeInfo {
+        block_size: 512,
+        volumes: Vec::new(),
+    };
     let mut ri: RegionInfo = RegionInfo {
-        block_size: 0,
-        extent_size: Block::new_512(0),
+        volume_info,
```

Comment:
Should we remove `total_size` and `total_blocks` here, because they can both be calculated from `volume_info`?

Reply:
Upon further reading, it would minimize the diff to also add a `block_size()` helper function, since lots of the diff in this PR is `s/block_size/volume_info.block_size`.

```rust
fn block_size(&self) -> u64 {
    self.volume_info.block_size
}
```

Reply:
I liked the idea of computing it once, then just passing it around, but don't really care if we compute it every time we need it.  I don't think its anywhere inside the performance path, and if I find a place I'll handle it there.

Reply:
I've updated the struct and added new methods on the type.